packages/chainsaw/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=chainsaw
+pkgver=18.9503898
+pkgrel=1
+pkgdesc='A powerful ‘first-response’ capability to quickly identify threats within Windows event logs.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-misc')
```

Comment:
Maybe add `blackarch-defensive`, `blackarch-forensic`, `blackarch-windows` and remove `blackarch-misc`. To list categories: https://gist.github.com/noraj/5045be32490f1161161e16fc5820066c#ba